src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -20,18 +20,26 @@
 /** The intake of the robot. */
 public class Intake extends SubsystemBase {
 
-  private CANSparkMax intake;
+  private CANSparkMax intakeLower;
+  private CANSparkMax intakeUpper;
   private DigitalInput limitSwitch;
 
   /** Creates a new Intake. */
   public Intake() {
-    this.intake = new CANSparkMax(Constants.Intake.INTAKE_MOTOR_ID, MotorType.kBrushless);
+    this.intakeLower =
+        new CANSparkMax(Constants.Intake.LOWER_INTAKE_MOTOR_ID, MotorType.kBrushless);
+    this.intakeUpper =
+        new CANSparkMax(Constants.Intake.UPPER_INTAKE_MOTOR_ID, MotorType.kBrushless);
     this.limitSwitch = new DigitalInput(Constants.Intake.INTAKE_LIMIT_SWITCH_DIO_PORT);
+
+    intakeLower.setInverted(Constants.Intake.LOWER_INTAKE_MOTOR_INVERTED);
+    intakeUpper.setInverted(Constants.Intake.UPPER_INTAKE_MOTOR_INVERTED);
   }
 
   /** Runs the intake motors. */
   public void runIntake() {
-    intake.set(Constants.Intake.INTAKE_MOTOR_SPEED);
+    intakeUpper.set(Constants.Intake.INTAKE_MOTOR_SPEED);
+    intakeLower.follow(intakeUpper, Constants.Intake.INTAKE_FOLLOWER_INVERSE_STATE);
```

Comment:
DO NOT run the follow call in a method that will be run by `execute()`, ESPECIALLY not after you call `.set(double)` on the motor it's following. `.follow(CANSparkMax, boolean)` changes a setting on the motor controller that tells it to listen to the CAN commands for the other motor, so you want to call it when you initialize the motors.